Makefile

```diff
@@ -29,11 +29,14 @@ all-tests::
 
 prepare:: prepare-external prepare-props
 
+# $(call GetPath,path)
+GetPath   = $(shell $(MSBUILD) $(MSBUILD_FLAGS) /p:DoNotLoadOSProperties=True /nologo /v:minimal /t:Get$(1)FullPath build-tools/scripts/Paths.targets | tr -d '[[:space:]]' )
```

Comment:
I'm not a fan of this kind of {x,ms}build invocation, to be honest. It's convenient if it works, but if it breaks... The error output of msbuild is mangled and next to impossible to read, also it gets interpreted by shell as a set of commands. Maybe (not as part of this PR) we should have a wrapper script for such one-liners that will properly check the exit status of the command as well as log and process its output.